staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -273,8 +273,8 @@ type ValidatingWebhook struct {
 	// +optional
 	ObjectSelector *metav1.LabelSelector `json:"objectSelector,omitempty" protobuf:"bytes,10,opt,name=objectSelector"`
 
-	// SideEffects states whether this webhookk has side effects.
-	// Acceptable values are: Unknown, None, Some, NoneOnDryRun
+	// SideEffects states whether this webhook has side effects.
```

Comment:
Would it be worth clarifying what happens when not specified (can it, and should we default it)?

Reply:
it cannot be omitted in v1. it's required and not defaulted

Reply:
AFAIU, the distinction between `None` and `NoneOnDryRun` makes no difference, so I'm not really sure why we force people to specify a value that adds no value?

Reply:
making it explicit in v1 ensures all manifests ack that no side effects are allowed. the distinction between None and NoneOnDryRun doesn't matter for the dispatcher, but allows expressing that there is a difference in behavior specifically for dry run